pkg/flow/states/parallel.go

```diff
@@ -345,7 +345,7 @@ func (logic *parallelLogic) processActionResults(ctx context.Context, children [
 		children[idx].Complete = true
 		completed++
 
-		logic.Log(ctx, "Action returned. (%d/%d)", completed, len(children))
+		logic.SendToLogger(ctx, "info", "Action returned. (%d/%d)", completed, len(children))
```

Comment:
`Log` was roughly renamed to `SendToLogger`, but I don't think that was necessary. `Log` should have just been changed to support log levels. Or maybe another function should have been added to the interface alongside it: `LogError`.